content/psychologists-recommend/confidence-acquirement.md

```diff
@@ -0,0 +1,43 @@
+title: Рекомендации по приобретению уверенности
+slug: confidence-acquirement
```

Comment:
Replace `slug` with `acquiring-confidence`. File name should be replaced with `acquiring_confidence`. See the difference.